packages/zowe-explorer/__mocks__/@zowe/imperative.ts

```diff
@@ -150,6 +150,17 @@ export class ProfileInfo {
     }
 }
 
+// tslint:disable-next-line:max-classes-per-file
```

Comment:
When https://github.com/zowe/vscode-extension-for-zowe/pull/2029 is merged, we may need to update this too 🙂